validation/barotropic_gyre/barotropic_gyre.jl

```diff
@@ -35,14 +35,16 @@ underlying_grid = LatitudeLongitudeGrid(size = (Nx, Ny, 1),
                                         latitude = (15, 75),
                                         z = (-4000, 0))
 
-@inline raster_depth(i, j) = 30 < i < 35 && 42 < j < 48
-
-grid = ImmersedBoundaryGrid(underlying_grid, GridFittedBoundary(raster_depth, mask_type=RasterDepthMask()))
+## bathymetry = zeros(Nx, Ny) .- 4000
+## view(bathymetry, 31:34, 43:47) .= 0
+## bathymetry = arch_array(arch, bathymetry)
```

Comment:
we _try_ to handle the architecture under the hood for you... (though maybe it's not working...)

Reply:
Yeah - I still need to debug that bit.